frontend/ui/elements/screen_fullscreen_menu_table.lua

```diff
@@ -24,6 +25,11 @@ return {
         local screen_height = android.getScreenHeight()
         logger.dbg("screen_fullscreen_menu_table.lua: Screen height: ", screen_height)
 
+        local new_height = screen_height - status_bar_height
+        logger.dbg("screen_fullscreen_menu_table.lua: Setting viewport to {x= 0, y=" .. status_bar_height ..", w=" .. screen_width .. ", h=" .. new_height .. "}")
```

Comment:
Good idea but I'd say wrong place. I don't think there's a relevant location in front where this'd go.

But it should go in base (`self.debug`) just before this line:
https://github.com/koreader/koreader-base/blob/adc7d32ced79f264138c0b26bc9d40446628c2da/ffi/framebuffer.lua#L144

I can take care of it if you like.

Reply:
@Frenzie  Great. Thanks:)

Reply:
I meant in base. Would you mind removing this line? :-P

Reply:
done